scalahost/nsc/src/test/scala/scala/meta/tests/scalahost/mirrors/SemanticSuite.scala

```diff
@@ -205,11 +205,11 @@ class SemanticSuite extends DatabaseSuite(SemanticdbMode.Slim) {
       |object RunMe {
       |  CommandeerDSL(null.asInstanceOf[Foo])
       |}
-  """.trim.stripMargin,
+    """.trim.stripMargin,
     """
-      |[324..324) [g.Foo, g.FooDSL]
-      |[348..348) (g.Foo.fooDSL)
-  """.trim.stripMargin
+      |[324..324) (*).apply[g.Foo, g.FooDSL]
+      |[348..348) (*)(g.Foo.fooDSL)
```

Comment:
Same question about `*(...)` and positioning the sugar at `CommandeerDSL(null.asInstanceOf[Foo])`.